sidebars.json

```diff
@@ -486,8 +486,19 @@
           "type": "doc"
         },
         {
-          "id": "console/design-your-projects/custom-resources",
-          "type": "doc"
+          "collapsed": true,
+          "link": {
+            "id": "console/design-your-projects/custom-resources/custom-resources",
```

Comment:
Are we moving the page from `https://docs.mia-platform.eu/docs/console/design-your-projects/custom-resources` to `https://docs.mia-platform.eu/docs/console/design-your-projects/custom-resources/custom-resources`?

If so, should we update the `301redirects.json` file?